cumulus_etl/etl/studies/irae/irae_tasks.py

```diff
@@ -0,0 +1,69 @@
+"""Define tasks for the irae study"""
```

Comment:
This file is worth calling out - because after all the rest of the refactoring here, this file is all that's needed to add a new study - basically just define the system & user prompts, define the output structure, and choose a supported NLP model.